mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/MobileVerificationActivity.java

```diff
@@ -105,12 +116,19 @@ public void onRequestOtpSuccess() {
         mFabNext.setVisibility(View.VISIBLE);
     }
 
+    /**
+     * This function show a toast when its fails to request an OTP
+     * @param s This is the message that is displayed on the toast.
+     */
     @Override
     public void onRequestOtpFailed(String s) {
         hideProgressDialog();
         showToast(s);
     }
 
+    /**
+     * This function is implemented when Next is clicked.
+     */
```

Comment:
The function signature is already verbal enough to signify 'Next is clicked'
Instead the documentation should be what does this do (it's hiding the keyboard and why the delay)